package.json

```diff
@@ -9,32 +9,35 @@
     "yarn": "1.22.11"
   },
   "scripts": {
-    "ng": "nx",
-    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
```

Comment:
Question: What is the reason for not running `ngcc` after install locally but including it in the pipeline?

Reply:
It's not required for certain tooling so I don't want to waste precious time or resources in jobs that don't require it. Additionally, it can be optimized further for single applications. Especially Angular CDK and Material are terrible with NGCC because of their dozens of entry point. Angular CLI will run NGCC on demand if we don't do it in postinstall. However, the ngcc script will be faster than default NGCC settings.